Tilemap migration in docs (#1623)
microsoft/pxt-arcade: docs/concepts/breadcrumb-trail.md

Find ``||scene:set tile map to||`` in ``||scene:Scene||``
and drag it into the ``||loops:on start||``.

> Open the tilemap editor and use the paint bucket to fill it with a grassy background.

Reviewer: ```suggestion
Open the tilemap editor and use the paint bucket tool to fill the canvas with a grassy background
```

    img`
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
        . . . . . . . . . . . . . . . .
    `,
    [myTiles.tile0,sprites.castle.tileDarkGrass3],
    TileScale.Sixteen
))

> s

Reviewer 2: ```suggestion
```

typo

}
```

## Step 6

Find ``||scene:place mySprite on top of tilemap col row||``
and place it after ``||sprites:set mySprite to||``.
Drag ``||variables:value||`` from the ``||loops:for loop||`` to ``||variables:tilemap col row||``.
Make sure ``||variables:mySprite||`` is the ``||variables:variable||`` for the
``||sprites:sprite||`` you just created.

> This will place the breadcrumbs on each of the ``||scene:tile||``s in the diagonal line.

Reviewer: ```suggestion
This will place the breadcrumbs on each ``||scene:tile||`` of the diagonal line.
```
